blocks/library/gallery/block.js

```diff
@@ -262,4 +310,40 @@ class GalleryBlock extends Component {
 	}
 }
 
-export default GalleryBlock;
+// FIXME
```

Comment:
The data module might help here since it's a canonical way to access the WP client state #3832 
But we can't assume the post is present, I'm thinking the gallery block should work for site building later and we don't have a post necessarily. This option should be only available if we detect an "editor" state and a "post Id".

Reply:
> #3832

I had started to review this and should’ve connected the dots. Thanks, this sounds like a good way to try the module. 

> can’t assume […] site building […] only if we detect 

Yes, great points!